src/packages/badge/badge.taro.tsx

```
@@ -77,48 +68,40 @@
+  const getCustomStyle = () => {
     const style: CSSProperties = {}
     if (color) {
       if (fill === 'outline') {
         style.color = color
         isHarmony
-          ? (style.backgroundColor = '#fff')
-          : (style.background = '#fff')
-        if (!color?.includes('gradient')) {
-          style.borderColor = color
-        }
+          ? (style.backgroundColor = '#FFFFFF')
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)